causalpy/plot_utils.py

```diff
@@ -13,28 +14,46 @@ def plot_xY(
     ax: plt.Axes,
     plot_hdi_kwargs: Optional[Dict[str, Any]] = None,
     hdi_prob: float = 0.94,
+    label: Optional[str] = "",
     include_label: bool = True,
-) -> None:
+):
     """Utility function to plot HDI intervals."""
 
     if plot_hdi_kwargs is None:
         plot_hdi_kwargs = {}
 
-    az.plot_hdi(
+    (h_line,) = ax.plot(
+        x,
+        Y.mean(dim=["chain", "draw"]),
+        ls="-",
+        **plot_hdi_kwargs,
+        label=f"{label}" if include_label else None,
+    )
+    ax_hdi = az.plot_hdi(
         x,
         Y,
         hdi_prob=hdi_prob,
         fill_kwargs={
             "alpha": 0.25,
-            "label": f"{hdi_prob*100}% HDI" if include_label else None,
+            "label": " ",  # f"{hdi_prob*100}% HDI" if include_label else None,
```

Comment:
shall we start removing commented code? this makes the code hard to maintain.

Reply:
Fixed in an upcoming commit